test/happy-defaults.test.js

```diff
@@ -0,0 +1,49 @@
+const packmap = require("packmap");
+const fsMock = require("fs-extra");
+
+jest.mock("../lib/require-resolve.js");
+
+describe(`happy defaults`, () => {
```

Comment:
Minor. IMHO Grouping tests based on functionality will be preferable than grouping by test type.

Reply:
I don't know what you mean by "test type" - my plan is to have a separate test file for a wide variety of use cases.